data/docs/pachyderm-beginner-tutorial.yaml

```diff
@@ -0,0 +1,15 @@
+metadata:
+  name: pachyderm-beginner-tutorial
+  type: tutorial
+  annotations:
+    opendatahub.io/categories: 'AI/Machine learning, Pipeline'
+spec:
+  appName: pachyderm
+  displayName: Getting started with Pachyderm concepts
+  description: |-
+    Learn Pachyderm's main concepts by creating pipelines that perform edge detection on a few images. 
+    Check the Getting Started section to install Pachyderm's CLI and connect to your cluster from your notebook.
```

Comment:
```suggestion
  
``` 
I recommend cutting the last line because the tutorial link already brings the user to the "Getting Started" section, specifically the "Beginner Tutorial" sub-section under "Getting Started." The tutorial document is organized in a very intuitive way anyways, so I don't recommend including additional information on the card about how to navigate the document.

Reply:
Sounds good; it makes complete sense. I got slightly confused with the navigation paths mainly because I was not able to serve the docs.